modelon/impact/client/sal/project.py

```diff
@@ -87,7 +87,7 @@ def fmu_upload(
         url = (
             self._base_uri
             / f"api/workspaces/{workspace_id}/projects/{project_id}/content/"
```

Comment:
And it ´does not contain workspaces/{workspace_id} anymore? That is the bigger change.